frontend/src/store.js

```diff
@@ -42,6 +42,10 @@ const store = createStore({
     setUser: (state, user) => {
       localStorage.setItem('user', JSON.stringify(user));
       state.user = user;
+    },
+    setConferenceRoomType: (state, conferenceRoomType) => {
+      state.conferenceRoomType = conferenceRoomType;
+      console.log('this is the state', conferenceRoomType);
```

Comment:
Let remove that log. So after that there is no need to a setter. Let's just use the state directly. Some say its bad practice, but this is an example app the public sees and it should be as straight forward, simple and clean as possible. We also don't use setters to set any of the other store states.